### PR TITLE
Refactor/find and replace assets zappy gui

### PR DESCRIPTION
**Pull request by mathieurobert1**
I changed the old and ugly assets by new ones and added some decoration

> [!IMPORTANT]
> With multi-threading we will be able to display lanterns ! (Because currently my computer want to die)

Screenshot from 2024-06-12 14-55-47

image

### CODE REVIEW

File: gui/include/Render/Decoration.hpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+#include <vector>
+#include <iostream>
+
+/**
+ * @brief Map type.
+ *
+ * @tparam T Type of the map.
+*/
+template<typename T>
+using Map = std::vector<std::vector<T>>;
```
Comment (Thomaltarix) [inline gui/include/Render/Decoration.hpp:22]: You should build a types.hpp file in my opinion since many files needs the same types

**Review by theophile-jr — DISMISSED** [review]

File: gui/src/Render/Decoration.cpp [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Gui::Decoration::Decoration()
+{
+    _treeModel = LoadModel(MODEL_TREE);
+    //TODO : Unlock when multithreading: _lanternModel = LoadModel(MODEL_LANTERN);
+}
+
+Map<bool> Gui::Decoration::getGenerationItem(std::size_t ratio)
+{
+    (void) ratio;
```
Comment (Thomaltarix) [inline gui/src/Render/Decoration.cpp:18]: It  might be deleted to work correctly x)

**Review by Njord201 — APPROVED** [review]

**Review by Thomaltarix — CHANGES_REQUESTED** [review]
Some changes, but overall great job

**Review by Njord201 — APPROVED** [review]

**Review by Thomaltarix — APPROVED** [review]
Clean one

**Review by theophile-jr — APPROVED** [review]

**Pull request merged**